package.json

```diff
@@ -135,6 +135,9 @@
     "commitmsg": "commitlint --edit $HUSKY_GIT_PARAMS",
     "build-storybook": "build-storybook -c .storybook -o docs",
     "lint": "node utils/check-lockfile-links.js && node utils/check-mismatched-dependencies.js && eslint -c ./.eslintrc.js --ext=jsx,ts,tsx .",
+    "typecheck": "yarn typecheck:src && yarn typecheck:cypress",
+    "typecheck:src": "tsc -p . --noEmit --incremental false",
```

Comment:
Won't this also do `cypress/` since that folder is in `./`?

Reply:
The `-p` means project where the path determines where the `tsconfig.json` file will be. That file ignores the `cypress` directory.

We had to split Cypress and the rest of the application up because both `jest` and `cypress` add globals with colliding definitions.

Reply:
Doh - haha sounds good.